src/app/components/EditorInterface.tsx

```diff
@@ -137,10 +183,36 @@ const EditorInterface = () => {
     }
   }, [editor])
 
+  useEffect(() => {
+    ;(async () => {
```

Comment:
is this a typo `;`

Reply:
This typo is coming because of prettier formatter. We may have to work on the formatter somehow. @saugatmaharjan

Reply:
seems like formatting with prettier automatically puts this